docs/api.list_database_connections.rst

```diff
@@ -29,6 +30,10 @@ HTTP 200 code response
         "use_ssh": true,
         "uri": null,
         "path_to_credentials_file": null,
+        "llm_credentials": {
```

Comment:
these are fake credentials correct?

Reply:
Yes, I got `cryptography.fernet.InvalidToken` when I tried to decrypt it